CMakeLists.txt

```diff
@@ -93,6 +93,13 @@ SET(${PROJECT_NAME}_EXCLUDE_DISABLED_SUBPACKAGES_FROM_DISTRIBUTION_DEFAULT FALSE
 
 SET(Trilinos_USE_GNUINSTALLDIRS_DEFAULT ON)
 
+SET(Trilinos_MUST_FIND_ALL_TPL_LIBS_DEFAULT TRUE)
+
+SET(CMAKE_LINK_LIBRARIES_ONLY_TARGETS ON)
+
+add_library(toolchain::m INTERFACE IMPORTED)
+set_property(TARGET toolchain::m PROPERTY IMPORTED_LIBNAME "m")
```

Comment:
My only concern is that this creates more clutter in the top CMakeLists.txt file and makes the packages less self-contained.  Why not just copy define two targets `Zoltan::m` and `SuiteSparse::m` and put them in the packages themselves?

Reply:
I'm willing to do that, since it keeps the packages more independently-buildable.  Stand by